molecule/filebeat/molecule.yml

```diff
@@ -5,6 +5,9 @@ driver:
   name: docker
 lint:
   name: yamllint
+  options:
```

Comment:
Why is this necessary?

Reply:
Hi Manuel,

This is needed because `yamllint` testing fails in virtual environments. 

Check this [issue](https://github.com/ansible/molecule/issues/1466#issuecomment-462476230) for more info.

Kind Regards,

Rshad

Reply:
Thanks